README.md

```diff
@@ -34,11 +34,15 @@
 
 ## Requirements
 
-Starting from version `0.23.2` of the library your project is required to use:
+Starting from version `0.23.3` of the library your project is required to use:
 
 * Gradle `7.0` or newer
```

Comment:
Below in this file: `Gradle configuration is supported for all platforms, minimal version is Gradle 6.8.`

Reply:
Thanks, didn't notice that. I'd remove this line completely.